src/common/string/filter/sequence-matching.ts

```diff
@@ -51,8 +54,7 @@ export const fuzzySequentialMatch = (filter: string, ...words: string[]) => {
 
 export interface ScorableTextItem {
   score?: number;
-  filterText: string;
-  altText?: string;
+  words: string[];
```

Comment:
I'm open to other names for this property. All the names I could think of didn't seem quite right. 

"words" isn't quite right cuz it's not a list of single words. It's a list of target strings that our filter will be trying to match against.

"filterStrings" maybe? Or "stringIdentifiers"?

Reply:
Just `strings`?

Reply:
I'm okay with just `strings`. It's just not very self-descriptive. I could rectify that in the commented method descriptor though.

Reply:
Changed to "strings" and adjust the tsdoc descriptions to help explain the parameters better.
![image](https://user-images.githubusercontent.com/1480827/114615922-1ac4e380-9c5b-11eb-96b0-de7ab541ed16.png)